server/utils/courseUtils.ts

```diff
@@ -45,6 +45,7 @@ const prerequisiteSummaryListRows = (prerequisites: Array<CoursePrerequisite>):
 const presentCourse = (course: Course): CoursePresenter => {
   return {
     ...course,
+    nameAndAlternateName: `${course.name} (${course.alternateName})`,
```

Comment:
as mentioned above, we'll want to only add the ` (${course.alternateName})` if the course has an acronym